constraint/pkg/apis/templates/v1alpha1/constrainttemplate_types.go

```diff
@@ -29,6 +29,11 @@ type ConstraintTemplateSpec struct {
 	Targets []Target `json:"targets,omitempty"`
 }
 
+type MutationTemplateSpec struct {
```

Comment:
mutationtemplate type should be created via `kubebuilder create`, which would put all of this in a separate directory.